authapi/permissions.py

```diff
@@ -19,7 +20,9 @@ def __init__(self, permission_type):
     def has_permission(self, permission, request, view):
         user = request.user
         permissions = get_user_permissions(user)
-        permissions = find_permission(permissions, self.permission_type)
+        permissions = find_permission(
+            permissions, self.permission_type,
+            namespace=settings.PERMISSION_NAMESPACE)
```

Comment:
Is there a simple way we could test that `AllowPermission` and `AllowObjectPermission` ignore permissions for a different namespace?